### DISCUSSION TITLE
Welcome to DeviceRunners Discussions!

### DISCUSSION THREAD

**Discussion by mattleibow** [mattleibow/DeviceRunners#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bonniemathew** [comment]
I have recently migrated our project from Xamarin.iOS and Xamarin.Android to .Net 8.0 iOS and .Net8.0 Android. I would like to know how to configure the DeviceRunner to work on the devices in the case of non-MAUI applications. I see a sample app on how to configure xUnit for Net Maui app. But how to configure the device runner and Link the test Assembly for .net8.0-ios /.net8.0-android applications?

**Reply by mattleibow** [reply]
Right now I have not written the UI for that since I have not had time.

**Comment by Rudhra-balan** [comment]
Hi, Im trying to use the device runner project unit test case for MAUI windows. I want to inculde the complete code coverage result . Is it possiable ?
